disnake/channel.py

```diff
@@ -1860,15 +1861,22 @@ async def create_instance(
             The stage instance's topic.
         privacy_level: :class:`StagePrivacyLevel`
             The stage instance's privacy level. Defaults to :attr:`StagePrivacyLevel.guild_only`.
-        reason: Optional[:class:`str`]
-            The reason the stage instance was created. Shows up on the audit log.
         notify_everyone: :class:`bool`
             Whether to notify ``@everyone`` that the stage instance has started.
             Requires the :attr:`~Permissions.mention_everyone` permission on the stage channel.
             Defaults to ``False``.
 
             .. versionadded:: 2.5
 
+        guild_scheduled_event: :class:`abc.Snowflake`
+            The guild scheduled event associated with the stage instance.
+            This will automatically start the guild scheduled event.
```

Comment:
Why not `Setting this will automatically start the guild scheduled event.` (apart from redundancy). Just to be consistent when mentioning Guild Scheduled Events.

Reply:
Just felt it's unnecessary since "guild scheduled event" is mentioned right before; the API docs use "guild scheduled event", "scheduled event", and "event" interchangeably.

Reply:
Alright